travis: windows build is failing on a couple of non-core modules

`internal/contributebot`:

```
# gocloud.dev/internal/contributebot [gocloud.dev/internal/contributebot.test]
.\sync.go:177:25: undefined: unix.SIGTERM
FAIL	gocloud.dev/internal/contributebot [build failed]
?   	gocloud.dev/internal/contributebot/webhook	[no test files]
```

We ran into this before and decided to just not run them. Maybe that's still the right answer, but now that we run all tests for all modules in a loop (instead of bespoke code for each), it's less obvious how to do that.


`internal/website`:

```
--- FAIL: TestGather (55.07s)
    --- FAIL: TestGather/BlankImports (0.19s)
        gatherexamples_test.go:265: go [list -e -json -compiled=true -test=true -export=false -deps=true -find=false -- ./...]: exit status 1: error writing go.mod: rename C:\Users\travis\AppData\Local\Temp\TestGather_BlankImports835612617\primarymod\foo\go.mod562915362.tmp C:\Users\travis\AppData\Local\Temp\TestGather_BlankImports835612617\primarymod\foo\go.mod: Access is denied.
            
FAIL
FAIL	gocloud.dev/internal/website/gatherexamples	55.428s
```

This just looks like a bug where a file is read and not closed before it's being written.

--- a/internal/testing/runchecks.sh
+++ b/internal/testing/runchecks.sh
@@ -96,7 +96,14 @@ while read -r path || [[ -n "$path" ]]; do
     fi
   else
     echo "Running Go tests..."
-    go test -mod=readonly -race ./... || result=1
+    # TODO(rvangent): Special case modules to skip for Windows. Perhaps
+    # this should be data-driven by allmodules?
+    # (https://github.com/google/go-cloud/issues/2111).
+    if [[ "${TRAVIS_OS_NAME:-}" == "windows" ]] && ([[ "$path" == "internal/contributebot" ]] || [[ "$path" == "internal/website" ]]); then
+      echo "  Skipping tests on Window"
+    else
+      go test -mod=readonly -race ./... || result=1
+    fi
   fi
   # Do these additional checks for the Linux build on Travis, or when running
   # locally.

